### PR TITLE
Sandbox logs to file

### PR DESCRIPTION
**Pull request by Atlante45**
Included a new package so we can save the logs to a file.
- on Linux: `~/.local/share/High Fidelity/Server Console/log.txt`
- on OS X: `~/Library/Application Support/High Fidelity/Server Console/log.txt`
- on Windows: `%USERPROFILE%/AppData/Roaming/High Fidelity/Server Console/log.txt`

Added debug to shutdown process.

Test Plan:
- Run the Sandbox for a little bit, close it.
- Check that the log file was correctly saved.

### CODE REVIEW

File: server-console/src/main.js [hunk 1]
```diff
@@ -74,10 +74,16 @@ function getBuildInfo() {
 
     return buildInfo;
 }
-
 const buildInfo = getBuildInfo();
 
-console.log("build info", buildInfo);
+const logFile = getApplicationDataDirectory() + '/log.txt';
```
Comment (Atlante45) [inline server-console/src/main.js]: It does. Just needs to have `buildInfo` defined before.

File: server-console/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "hf-console",
```
Comment (Atlante45) [inline server-console/package.json]: @murillodigital I'm not sure if this is used on the Jenkins side.
Couple names that hadn't been ported over to Sandbox.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4661d6e2d12c979e04edf8c86a2e0ba337a42cbf available at: https://gist.github.com/44d0e819b66ceb819140846c84d06d18<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8892-4661d6e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8892-4661d6e.dmg

**Comment by ctrlaltdavid** [comment]
Windows 10
High Fidelity build 5429 installed but Sandbox and Interface not running.
Downloaded and installed PR build, electing to copy settings etc. from existing install.
Started PR sandbox.
Immediately got error dialog, attached.
"OK" to error dialog; server icon not in Windows tray but 2 x "Sandbox" processes listed in Task Manager.

Manually terminated the two Sandbox processes and tried again. Same error.

pr8892 error

**Comment by Atlante45** [comment]
@ctrlaltdavid Arg, it's a real chicken and egg problem. Dev "build" didn't seen to have issue with it.
Re-organizing the code, I should have a new build soon.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 812265363c9c0e79dd960f18bb5596539fba6865 available at: https://gist.github.com/44d0e819b66ceb819140846c84d06d18<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8892-8122653.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8892-8122653.dmg

**Comment by ctrlaltdavid** [comment]
Retested:
- Uninstalled previous PR install and deleted copied settings and data files before installing new PR build.
- Sandbox started up OK this time.
- Visited Sandbox (localhost).
- Exited Interface and Sandbox.
- Log file written as expected.
- Restarting Sandbox appends to log file.

Not tested: Any roll over of log.txt file.

Note: Windows tray icon's "View Logs" menu item opens "Log" dialog but the new log.txt file is not directly accessible in a tab or via the "Open Log Directory" button (have to go up a level to see it). Perhaps log.txt should be created in the \logs directory?

**Comment by Atlante45** [comment]
The logs folder is meant for DS and AC logs. This one is slightly different and does not directly relate to the Domain's operations, so I put it at the root of the `Server Console` directory.

**Pull request merged**